Sources/MessagingPush/RichPush/RichPushRequestHandler.swift

```diff
@@ -21,7 +22,11 @@ internal class RichPushRequestHandler {
         let existingRequest = requests[requestId]
         if existingRequest != nil { return }
 
-        let diGraph = DIGraph.getInstance(siteId: siteId)
+        let sdkInitializedUtil = SdkInitializedUtilImpl()
+
+        guard let postSdkInitializedData = sdkInitializedUtil.postInitializedData else { return }
+
+        let diGraph = postSdkInitializedData.diGraph
```

Comment:
This is the new (safe) way to get the DiGraph instance. You see other files in this PR also doing it this way.